ewasm-testnet-cpp-config.json

```diff
@@ -21,14 +21,14 @@
     "allowFutureBlocks" : false 
   }, 
   "genesis": { 
-    "nonce": "0x0000000000000042", 
-    "difficulty": "0x400", 
+    "nonce": "0x0000000000000000", 
```

Comment:
Why? :(

Reply:
I'm assuming this was 0x42 to prevent some kind of replay attack?  All these changes were made to try and add a genesis file that was compatible with the one i am currently using on the testnet for geth.  Should probably change that and add the `extraData` and `nonce` values that are used here.

Reply:
Yeah I am not sure what this field was for, but the `nonce` isn't zero in the mainnet genesis, iirc.

Reply:
It was zero in the genesis that puppeth created.